lib/matplotlib/mathtext.py

```diff
@@ -467,19 +464,8 @@ def set_canvas_size(self, w, h, d):
 
     def render_glyph(self, ox, oy, facename, font_class, sym, fontsize, dpi):
         """
-        Draw a glyph at
-
-          - *ox*, *oy*: position
-
-          - *facename*: One of the TeX face names
-
-          - *font_class*:
-
-          - *sym*: TeX symbol name or single character
-
-          - *fontsize*: fontsize in points
-
-          - *dpi*: The dpi to draw at.
+        At position (*ox*, *oy*), draw the glyph specified by the remaining
+        parameters (see `get_metrics` for their detailed description).
```

Comment:
But `facename` is not described in `get_metrics`.

Reply:
that's because it should have been called `font`, like everywhere else... `@cbook._rename_parameter`'d it.